src/components/history/History.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const History = ({ historyArray }) => {
+    
+  const historyList = historyArray.map(item => {
+    return (
+      <li key={Math.random()}>
```

Comment:
don't use `Math.random` as a key: https://reactjs.org/docs/lists-and-keys.html#keys

If anything use the index from the array:

```js
historyArray.map((item, i) => (
  <li key={i}>.........
));
```